audio/descriptors.c

```diff
@@ -26,10 +28,13 @@ __code __at(DSCR_AREA) struct usb_descriptors code_descriptors = {
         .bLength            = USB_DT_DEVICE_SIZE,
         .bDescriptorType    = USB_DT_DEVICE,
         .bcdUSB             = USB_BCD_V20,
-        .bDeviceClass       = 0, // Class and protocol defined per interface
+        /* Class defined per interface */
+        .bDeviceClass       = 0, 
         .bDeviceSubClass    = 0,
+        /* Protocol defined per interface */
         .bDeviceProtocol    = 0,
-        .bMaxPacketSize0    = 64, // kB
+        /* packet size is in bytes */
+        .bMaxPacketSize0    = 64,
```

Comment:
Why are you using 64 byte max packet size - can you add a comment here explaining why...

Reply:
Don't know of a technical reason, except that is what the logitech reference used... This is just ep0 though

Reply:
Put a comment `/* FIXME: Unsure why we are using 64 bytes, copied from a logitech XYZ device. */`

Reply:
Found it - it is the maximum size of a control packet, which is what EP0 handles. Added comment